at_onboarding_flutter/lib/at_onboarding_result.dart

```diff
@@ -0,0 +1,45 @@
+enum AtOnboardingResultStatus {
+  success, //Authenticate success
+  error, //Authenticate error
+  cancel, //User canceled
+}
+
+class AtOnboardingResult {
```

Comment:
Can we please have one line comments for `AtOnboardingResult` and its members as well.

Reply:
Fixed